dev_utils/run_integration_test_standalone.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/sh
+
+start_time=$( date -u +'%FT%TZ' )
+
+for cmd in s3cmd jq openssl
+do
+    if ! command -v "$cmd" >/dev/null 2>&1
+    then
+        printf '%s could not be found\n' "$cmd" >&2
+        exit 1
+    fi
+done
+
+infile=${1:-$(dirname "$0")/dummy_data.c4gh}
+if [ ! -f "$infile" ]; then
+    printf 'Unable to find regular file "%s"\n' "$infile" >&2
+    exit 1
+fi
+infile=$( realpath "$infile" )
+
+cd "$(dirname "$0")" || exit
+
+# we need this certificate to be of 600 to work with db connection
+chmod 600 certs/client-key.pem
```

Comment:
Should there be a check that the certificates exist? might be good to either have a user notice to run `make_certs.sh`, or to just have the script run it if the cert is missing.